src/Sylius/Bundle/ResourceBundle/composer.json

```diff
@@ -28,19 +28,21 @@
         "symfony/form":                    "~2.3",
         "sylius/resource":                 "0.14.*@dev",
         "sylius/storage":                  "0.14.*@dev",
-        "sylius/translation-bundle":       "0.14.*@dev",
         "friendsofsymfony/rest-bundle":    "1.5.*@dev",
         "jms/serializer-bundle":           "~0.13",
         "white-october/pagerfanta-bundle": "~1.0",
         "doctrine/doctrine-bundle":        "~1.3",
-        "willdurand/hateoas-bundle":       "@stable"
+        "willdurand/hateoas-bundle":       "@stable",
+        "stof/doctrine-extensions-bundle": "~1.1"
```

Comment:
IMO as this is merge, it's the topic of that PR too, as you add additional dependencies, where in fact that bundle was always "small base" with _limited_ deps. I guess this should be checked we need this or not.